THANKS.txt

```diff
@@ -147,6 +147,7 @@ Irvin Probst (ENSTA Bretagne) for pole placement.
 Ian Henriksen for Cython wrappers for BLAS and LAPACK
 Fukumu Tsutsumi for bug fixes.
 J.J. Green for interpolation bug fixes.
+Alex Wagner for rewriting and improving the Mann-Whitney-Wilcoxon test (stats.mww).
```

Comment:
Once the name of the function settles, this will need to be changed, here and in the release notes.